gradle/lib/dependabot/gradle/update_checker/version_finder.rb

```diff
@@ -99,10 +97,10 @@ def filter_ignored_versions(possible_versions)
           filtered = possible_versions
 
           ignored_versions.each do |req|
-            ignore_req = Gradle::Requirement.new(parse_requirement_string(req))
+            ignore_reqs = Gradle::Requirement.requirements_array(req)
             filtered =
               filtered.
-              reject { |v| ignore_req.satisfied_by?(v.fetch(:version)) }
+              reject { |v| ignore_reqs.any? { |r| r.satisfied_by?(v.fetch(:version)) } }
```

Comment:
We test both ruby and gradle requirement syntax here: https://github.com/dependabot/dependabot-core/blob/b0416b0f5287620dd40683f2860576f2b0b989fa/gradle/spec/dependabot/gradle/update_checker/version_finder_spec.rb#L118-L128